backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc

```diff
@@ -0,0 +1,11 @@
+[id="oadp-features-plugins"]
+= OADP features and plugins
```

Comment:
GLOBAL: s/plugin/plug-in 

Except!!! I'm inclined to leave this as "plugin" in filenames.